pagopa/pn-frontend
# Feat/pn 2391 refactoring footer links

## Short description
Separate links for Privacy Policy and Terms of Service

## List of changes proposed in this pull request
- Separate links in Footer component in pn-common
- Add prop to show or hide terms of service link
- Add optional props to inject urls for some links

## How to test
Start pn-pa-webapp or pn-personafisica-webapp, both links should be visible and should bring to the routes `/informativa-privacy` and `/termini-di-servizio`.
Start pn-landing-webapp or pn-personafisica-login, only the link for privacy policy should be visible with route `/informativa-privacy`
